drivers/hmis/app/graphql/types/hmis_schema/unit.rb

```diff
@@ -10,9 +10,25 @@ module Types
   class HmisSchema::Unit < Types::BaseObject
     field :id, ID, null: false
     field :name, String, null: true
-    field :beds, [HmisSchema::Bed], null: false
-    field :bed_count, Integer, null: false
     field :start_date, GraphQL::Types::ISO8601Date, null: false
     field :end_date, GraphQL::Types::ISO8601Date, null: true
+    field :unit_type, Types::HmisSchema::UnitTypeObject, null: true
+    field :project, Types::HmisSchema::Project, null: true
+    field :date_updated, GraphQL::Types::ISO8601DateTime, null: false
+    field :date_created, GraphQL::Types::ISO8601DateTime, null: false
+    field :user, HmisSchema::User, null: false
+    field :unit_size, Integer, null: true
+
+    def user
+      user = object.user
+      user.hmis_data_source_id = current_user.hmis_data_source_id
+      Hmis::Hud::User.from_user(user)
+    end
+
+    def unit_size
+      return object.unit_size if object.unit_size.present?
+
+      object.unit_type&.unit_size
+    end
```

Comment:
This is totally fine as is, but could also be written this way:
```suggestion
    def unit_size
     object.unit_size || object.unit_type&.unit_size
    end
```

Reply:
I didn't want to rule out `0` being a valid size

Reply:
Ooh, that's ugly. 
```
irb(main):001:0> tt = 0
=> 0
irb(main):002:0> tt | 5
=> 5
```